tests/sles4sap/publiccloud/qesap_ansible.pm

```diff
@@ -54,6 +54,19 @@ sub run {
             '/tmp/results/',
             '-type', 'f',
             '-iname', "*.xml");
+        my $find_ansible_known_issue = qr/\[OSADO\]\[softfail\] ([a-zA-Z]+#\S+) (.*)/;
+        my $ansible_output = script_output("cat $ret[1]");
+        my $reference;
+        my $desc_known_issue;
+
+        foreach my $ansible_line (split /\n/, $ansible_output) {
+            chomp $ansible_line;
+            if ($ansible_line =~ $find_ansible_known_issue) {
+                $reference = "bsc#$1";
```

Comment:
```suggestion
                $reference = $1;
```

The reference to which ticketing system will come from external directly, from Ansible.

Reply:
ok